examples/molecular-photoswitches/photoswitches.csv

```diff
@@ -0,0 +1,99 @@
+,SMILES,rate of thermal isomerisation from Z-E in s-1,Solvent used for thermal isomerisation rates,Z PhotoStationaryState,E PhotoStationaryState,E isomer pi-pi* wavelength in nm,Extinction,E isomer n-pi* wavelength in nm,Extinction coefficient in M-1 cm-1,Z isomer pi-pi* wavelength in nm,Extinction.1,Z isomer n-pi* wavelength in nm,Extinction coefficient in M-1 cm-1.1,Wiberg index,E-Z irradiation wavelength in nm,Z-E irradiation wavelength,Irradiation solvent
```

Comment:
Local copy of the photoswitch dataset from:

https://github.com/Ryan-Rhys/Photoswitches